src/hotspot/share/opto/type.cpp

```diff
@@ -710,6 +713,10 @@ void Type::Initialize_shared(Compile* current) {
   mreg2type[Op_VecY] = TypeVect::VECTY;
   mreg2type[Op_VecZ] = TypeVect::VECTZ;
 
+  LockNode::lock_type_init();
```

Comment:
Would be nice to have consistent naming here (e.g., `initialize_types` or even `initialize_c2_types`).

Reply:
I have updated name, can you take another look if they look fine now ? Thanks